extensions/objects/models/objects.py

```diff
@@ -1709,3 +1710,23 @@ class TranslatableSetOfUnicodeString(BaseTranslatableObject):
         'contentId': None,
         'unicodeStrSet': [],
     }
+
+
+class ValidateAndNormalizeJsonStrings(BaseObject):
```

Comment:
I still don't think this name is ideal, although I agree that making it more generic makes sense (using JSON instead of a list). I think the name should be `JsonString` or `JsonEncodedInString`.